math/fibonacci_sum.cpp

```diff
@@ -0,0 +1,135 @@
+/**
+ * @file
+ * @brief An algorithm to calculate the sum of Fibonacci Sequence: \f$\mathrm{F}(n) + \mathrm{F}(n+1) + .. + \mathrm{F}(m)\f$
```

Comment:
Provide a Wikipedia link in Markdown format (if available).
If there's no Wikipedia link, add another web source for algorithm explanation. 🙂

Reply:
Added.

Reply:
> Added.

The link should be added to the brief
```suggestion
 * @brief An algorithm to calculate the sum of [Fibonacci Sequence](https://en.wikipedia.org/wiki/Fibonacci_number): \f$\mathrm{F}(n) + \mathrm{F}(n+1) + .. + \mathrm{F}(m)\f$
```